pydfy/template/src/partials/kpi.html

```diff
@@ -0,0 +1,10 @@
+<div class="pf-kpi border-solid border border-gray-100">
+    <div class="pf-kpi-title bg-gray-200 text-lg font-bold text-center align-top">{{ component.title }}</div>
+    <div class="pf-number-content mt-2 text-center">
```

Comment:
This would also need to change to:
```suggestion
    <div class="pf-kpi-content mt-2 text-center">
```

Reply:
I figured as much, although I wasn't entirely sure if it was fine to leave colored black. To resolve this I have now added this class to blue.css to reflect that it is in fact a different component.

Reply:
Cool, at least makes the behavior explicit :+1: